PayPal/src/main/java/com/braintreepayments/api/PayPalInternalClient.java

```diff
@@ -53,6 +47,17 @@ public void onResult(@Nullable final Configuration configuration, @Nullable Exce
                                         ? SETUP_BILLING_AGREEMENT_ENDPOINT : CREATE_SINGLE_PAYMENT_ENDPOINT;
                                 String url = String.format("/v1/%s", endpoint);
 
+                                String cancelUrl;
+                                String successUrl;
+                                if (payPalRequest.isAppLinkEnabled() && braintreeClient.getAppLinkReturnUri() != null) {
+                                    String appLink = braintreeClient.getAppLinkReturnUri().toString();
+                                    cancelUrl = String.format("%s/cancel", appLink);
+                                    successUrl = String.format("%s/success", appLink);
+                                } else {
+                                    cancelUrl = String.format("%s://onetouch/v1/cancel", braintreeClient.getReturnUrlScheme());
+                                    successUrl = String.format("%s://onetouch/v1/success", braintreeClient.getReturnUrlScheme());
```

Comment:
Is there anything special about these `onetouch/v1` paths in the deep link URLs? Would we need these in the app link URLs or can we just append `/cancel` and `/success`?

Reply:
I don't think the onetouch naming is doing anything, it works with any random string in there.

Reply:
The gateway team confirmed that the `onetouch/v1` path is an SDK specific detail. So we're safe to omit it from the app link URL 👍